docs/api/color.md

```diff
@@ -0,0 +1,14 @@
+# color (string)
+
+A single hexidecimal color value or a comma sperated list of key value pairs. Keys must be one of the following `default`, `dark`, `aerial`, `deuteranopia` or `tritanopia` and values must be hexidecimal colors, eg. `#d4351c`.
```

Comment:
is this true - are we reverting to a hard coded limited set of styles?
FMfP doesn't use those values, we made it so the implementation could specify the keys.